test/ServerComparison.FunctionalTests/Helpers.cs

```diff
@@ -1,17 +1,34 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
 using System.IO;
 using Microsoft.AspNetCore.Server.Testing;
+using Microsoft.Extensions.PlatformAbstractions;
 
 namespace ServerComparison.FunctionalTests
 {
     public class Helpers
     {
         public static string GetApplicationPath(ApplicationType applicationType)
         {
-            return Path.GetFullPath(Path.Combine("..",
-            applicationType == ApplicationType.Standalone? "ServerComparison.TestSites.Standalone" : "ServerComparison.TestSites"));
+            var applicationBasePath = PlatformServices.Default.Application.ApplicationBasePath;
+
+            var directoryInfo = new DirectoryInfo(applicationBasePath);
```

Comment:
We should probably commonize this. There's like 3 copies of this method floating around.